src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -362,6 +363,55 @@ const getClickConversionPayloadAndEndpoint = (message, Config, filteredCustomerI
   return { payload, endpoint };
 };
 
+const GOOGLE_ALLOWED_CONSENT_STATUS = ['UNSPECIFIED', 'UNKNOWN', 'GRANTED', 'DENIED'];
```

Comment:
Please make sure to not repeat this for all the destinations.

Reply:
done